### ISSUE TITLE
Better error for 'does not export [named export]'

### ISSUE THREAD

**Issue by Rich-Harris** [rollup/rollup#1033]
A lot of people have been confused by messages like this:

> Module /path/to/node_modules/thing/foo.js does not export foo

In particular this has been causing a lot of pain for people who have been introduced to Rollup via Ionic in the last couple of weeks.

When the module in question is a CommonJS module like this...

```js
var myLib = exports;
myLib.foo = 'good luck trying to statically analyse this';
```

...it's very confusing. The error message should be changed to something that guides people towards the correct solution (without assuming it's CommonJS-related).

**Comment by thisconnect** [comment]
Does rollup know that this is CommonJS code?

**Comment by Rich-Harris** [comment]
No. By the time that error arises, it's just another ES module. We could conceivably add some sort of 'marker' (e.g. a comment) to the file as it passes through the plugin, but there'd be no guarantee it wouldn't be changed or removed by a subsequent transform. I think the easiest thing would be to link to the [Troubleshooting](https://github.com/rollup/rollup/wiki/Troubleshooting) page and have a section there explaining likely causes.

**Comment by Rich-Harris** [comment]
fixed in 0.36.2
